app/components/hiring_staff/inline_components/vacancies_count_heading_component.rb

```diff
@@ -0,0 +1,17 @@
+class HiringStaff::InlineComponents::VacanciesCountHeadingComponent < ViewComponent::Base
+  def initialize(vacancies: vacancies, selected_type: selected_type)
+    @vacancies = vacancies
+    @selected_type = selected_type
+  end
+
+  def call
+    case @vacancies.count
+    when 0
+      t("jobs.#{@selected_type}_jobs_with_count.zero")
+    when 1
+      t("jobs.#{@selected_type}_jobs_with_count.one")
+    else
+      t("jobs.#{@selected_type}_jobs_with_count", count: @vacancies.count)
```

Comment:
This alone is sufficient, the translation uses the count to decide which one to use itself